src/core/client/stream/components/Permalink/PermalinkView.tsx

```diff
@@ -0,0 +1,73 @@
+import qs from "query-string";
+import * as React from "react";
+import { StatelessComponent } from "react";
+
+import Logo from "talk-stream/components/Logo";
+import { Button, Flex, Typography } from "talk-ui/components";
+import CommentContainer from "../../containers/CommentContainer";
+import * as styles from "./PermalinkView.css";
+
+export interface InnerProps {
```

Comment:
Call it `PermalinkViewProps`? We usually used `InnerProps` when we use HOCs but we don't here.